### PR TITLE
Building with responsive design - TNW website clon

### PR DESCRIPTION
**Pull request by mricanho**
I use 2 breakpoints for this project, one at 768px and the other at 1024px.

I use float, flex-box, and grid for the sections.

I tried to reduce the DIV's in HTML for clean code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
...
+<p>Contributions, issues, and feature requests are welcome!<p>
+<p>Feel free to check the <a href="https://github.com/mricanho/Building-with-Responsive-Design/issues/2">issues page.</a></p>
+ 
+<h2>Show your support</h2>
+<p> Give a 
+  <g-emoji class="g-emoji" alias="star" fallback-src="https://github.githubassets.com/images/icons/emoji/unicode/2b50.png"><img class="emoji" alt="star" height="20" width="20" src="https://github.githubassets.com/images/icons/emoji/unicode/2b50.png"></g-emoji>
+  if you like this project!</p>
+  
+<h2>License</h2>
+  <p>This project is <a href="../Building-with-Responsive-Design/LICENSE">MIT</a> licensed.</p>
```
Comment (descholar-ceo) [inline README.md:38]: [Optional] Try to use markdown language in your readme file

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @mricanho   :handshake: :handshake: :handshake: 

# :+1:  Highlights :champagne: :champagne: :champagne: 
:+1: This is good to make it this far, your clone looks like the original website and it is really nice, congratulations on the good work you have done so far and your project is really almost done. Just some touches remaining.
:+1: Your repo looks professional, with very nice PR description, professional readme file and good commit messages, keep it up :muscle:
:+1: You used mobile first approach
:warning: Read the following changes required
# :red_circle: To improve: [Required Changes]
- [x] Latest funding section's header, try to make the right side looks like the original (large screens)

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-11-09 20-57-12 | Screenshot from 2020-11-09 20-57-55|
:point_right: Could you see the difference :question: :question: :question: 


- [x] Latest funding section's header, try to make the right side looks like the original (medium screens)

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-11-09 20-59-58| Screenshot from 2020-11-09 21-00-47 |
:point_right: Could you see the difference :question: :question: :question: 

- [x] Fix the footer on mobile screens

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-11-09 21-03-22| Screenshot from 2020-11-09 21-02-38|
:point_right: Could you see the difference :question: :question: :question: 

- [x] The usage of `!important` is highly discouraged
- [x] Improve the CSS best practices: (Don't repeat yourself = DRY) [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright
 __You have used (in your css):__ 
```css
height: 80px; /*=> 4 times*/


width: 100%; /*=> 33 times*/


display: flex; /*=> 63 times*/


display: flex;
flex-direction: column; /*=> 10 times*/
```

====> Advice you can create utility classes which have those properties you want to apply on elements and use them on those elements
Here is the [article which can help you](https://speckyboy.com/good-bad-css-practices/)

# :yellow_circle: Nice to have: [Optional changes]
- [ ] Reduce the width of thenextweb icon which appears on mobile screens
- [ ] Reduce the height of the three first articles on the home pages

Reviewed by [:man_technologist:  descholar-ceo](https://github.com/descholar-ceo)
_slack:_ `@descholar` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] Keep code DRY as suggested by previous TSE with utilitary selectors. 

> ## Using the Same Rule Repetitively
> 
> A lot of beginners use repetitive rules for each element. If you want to create a rule for multiple elements, use a class attribute instead.
> 
> For instance, instead of repeating color: blue rule for every tags and ID:
> ```
> #intro1 {color: #3498db; font-size: 10px; font-weight: bold;}
> 
> header {color: #3498db; font-size: 20px; background-color: green;}
> 
> #banner {color: #3498db; font-size: 30px; background-image: url(images/static.jpg);}
> ```
> Use a class instead:
> 
> ```
> .blue {color: #3498db;}
> ```



## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Change Required :vertical_traffic_light: :warning: 
## Hello @mricanho  :handshake: :handshake: :handshake: 
:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
## Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Please fix the heading of latest finding section on both large screens and medium screens, study the following screenshots to understand more about this issue:
#### Large screens:
| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-11-10 00-56-35 | Screenshot from 2020-11-10 00-58-58 |

:point_right: Could you see the difference :question: :question: :question: 

#### Medium screens
| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-11-10 01-04-53|Screenshot from 2020-11-10 01-05-50|
:point_right: Could you see the difference :question: :question: :question: 

If anything seems to be unclear to you, don't hesitate to contact me for more clarification slack: `@Mugirase Emmanuel` | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Review by layzrd — APPROVED** [review]
### Status: Approved 🙌 🎇
Hi @mricanho , Congratulations 🎊 👏, you did very well on this project.  Kindly fit the following optional enhancements.

[Optional]

- [ ] On the pull request, please specify all the breaking point you used in the project.

Now, you can move ahead.

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/)

**Pull request merged**